requirements.txt

```diff
@@ -3,3 +3,4 @@ ansible>=1.9.1,<2.0
 pytest==2.7.1
 PyYAML==3.11
 Sphinx==1.2.3
+ansible-lint
```

Comment:
We should always fix some kind of version requirement, just in case another project makes backwards-incompatible changes.